File: ProjetoCs-Reserva_de_Hotel-main/ReservaDeHotel/Models/Dono.cs
using System.ComponentModel.DataAnnotations;

namespace ReservaDeHotel.Models;
public class Dono
{
    [Key]
    public int DonoId {get;set;}
    public string? Nome { get; set;}
    public int NumeroTelefone { get; set;}
    public string? Email { get; set;}
    public string? Login { get; set;}
    public string? Senha { get; set;}
    public List<Hotel>? ListaHoteis { get; set;}

    public Dono()
    {
        ListaHoteis = new List<Hotel>();
    }
}

File: ReservaHotelFront/src/app/Dono.ts
import { Hotel } from './Hotel';

export class Dono {
    DonoId: number = 0;
    Nome: string | null = null;
    NumeroTelefone: number = 0;
    Email: string | null = null;
    Login: string | null = null;
    Senha: string | null = null;
    ListaHoteis: Hotel[] = [];
    Hotel: Hotel | null = null;

    constructor() {
        this.ListaHoteis = [];
    }
}
